addon/styles/_frost-link.scss

```diff
@@ -214,10 +215,69 @@ $frost-color-link-focus-box-shadow: 0 1px 6px rgba(0,158,239, 0.75);
     }
 
     .svg {
-      margin-left: 10px;
+      &:not(.action) {
```

Comment:
I considered going with `design` but decided against it at the end...  But I suppose in the long term we might have additional custom designs for a link.  But it might have to wait till Monday.